vendor/modules.txt

```diff
@@ -922,7 +926,7 @@ k8s.io/csi-translation-lib
 k8s.io/klog
 # k8s.io/kube-openapi v0.0.0-20190718094010-3cf2ea392886
 k8s.io/kube-openapi/pkg/util/proto
-# k8s.io/kubernetes v1.15.2
```

Comment:
Can we update the version in edgecore config file to 1.15.3 now...

Reply:
done